test/k8sT/Verifier.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Authors of Cilium
```

Comment:
For `git blame`'s sake, have you tried `git mv test/runtime/verifier.go test/k8sT/Verifier.go`? (I understand if git is not able to consider this a rename due to the many changes that there made to this file.)

Reply:
I just tried... without success. Unless I keep the rename into a separate commit (which would break git bisect), git isn't able to keep track of the renaming. That file was almost entirely rewritten so maybe that's okay?

Reply:
Yeah, it was worth a shot.